src/routes/safe/store/actions/transactions/utils/transactionHelpers.ts

```diff
@@ -333,14 +333,25 @@ export const mockTransaction = (tx: TxToMock, safeAddress: string, state: AppRed
     isSuccessful: null,
     modified: submissionDate,
     origin: null,
-    safe: safeAddress,
+    safe: '',
     safeTxHash: null,
     signatures: null,
     submissionDate,
     transactionHash: null,
     confirmations: [],
-    ...tx,
+    baseGas: 0,
+    gasPrice: '',
+    gasToken: '',
+    operation: 0,
+    refundReceiver: '',
+    safeTxGas: 0,
+    to: '',
+    value: '',
   }
+}
+
+export const mockTransaction = (tx: TxToMock, safeAddress: string, state: AppReduxState): Promise<Transaction> => {
+  const transactionStructure: TxServiceModel = { ...getMockTransactionServiceModel(), ...tx }
```

Comment:
why do we need to call `getMockTransactionServiceModel` if we already have a mocked transaction?

Reply:
which one?